### DISCUSSION TITLE
Can I install iraf in Msys2/MinGW?

### DISCUSSION THREAD

**Discussion by youngastronomy** [iraf-community/iraf#340] [Q&A] [answered]
Can I install the Iraf in Msys2/MinGW？ 

While compile:
```
/bin/sh: -c: line 1: syntax error near unexpected token `('
/bin/sh: -c: line 1: `if [ bin.Unable to determine platform architecture for (msys_nt-10.0-22631). !
```

**Comment by olebole** [comment, marked as answer]
The supported architectures are listed in the [installation instructions](https://iraf-community.github.io/install.html). On Windows, the recommended way is currently to use the "Windows Subsystem for Linux" with Debian or Ubuntu. In these distributions, IRAF is already pre-packaged and can be installed with "apt install iraf".

If you however make a patch to support other architectures, like MinGW, please create a Pull Request.
